### ISSUE TITLE
Show warning for invalid packages

### ISSUE THREAD

**Issue by davidhesselbomvcc** [alanhamlett/pip-update-requirements#36]
Given the requirements file `req.txt`, with the following contents:

```
non-existent-package
```

The command

```pip install -r req.txt``` produces the following output:

```
ERROR: Could not find a version that satisfies the requirement non-existent-package (from -r req.txt (line 1)) (from versions: none)
ERROR: No matching distribution found for non-existent-package (from -r req.txt (line 1))
```

However, `pur -f -r req.txt` (version 0.5.4) just says `All requirements up-to-date.` and exits with code `0`. I'd say this is a bug?

**Comment by davidhesselbomvcc** [comment]
@alanhamlett Does the new title you set for this issue mean that you intend to keep exiting with code `0` when this happens, even though when warning is issued?

**Comment by alanhamlett** [comment]
Yes, we can add a new flag to exit non-zero but by default I don't want to start exiting with an error when it wasn't doing that previously.

**Comment by davidhesselbomvcc** [comment]
> Yes, we can add a new flag to exit non-zero but by default I don't want to start exiting with an error when it wasn't doing that previously.

Fair enough

**Comment by alanhamlett** [comment]
Implemented with e11bae658f8c7e781230c43c0a4a465547c9e181 and released in v7.2.0.
